RELEASES.md

```diff
@@ -1,3 +1,266 @@
+Version 1.32.0 (2019-01-17)
+==========================
+
+Language
+--------
+#### 2018 edition
+- [You can now use the `?` operator in macro definitions.][56245] The `?`
+  operator allows you to specify zero or one repetitions similar to the `*` and
+  `+` operators.
+
+#### All editions
+- [You can now match against `PhantomData<T>` types.][55837]
+- [You can now match against literals in macros with the `literal`
+  specifier.][56072] This will match against a literal of any type.
+  E.g. `1`, `'A'`, `"Hello World"`
+- [Self can now be used as a constructor and pattern.][56365] E.g. 
+  ```rust
+  struct Point(i32, i32);
+
+  impl Point {
+      pub fn new(x: i32, y: i32) -> Self {
+          Self(x, y)
+      }
+
+      pub fn is_origin(&self) -> bool {
+          match self {
+              Self(0, 0) => true,
+              _ => false,
+          }
+      }
+  }
+  ```
+- [Self can also now be used in type definitions.][56366] E.g.
+  ```rust
+  enum List<T>
+  where
+  Self: PartialOrd<Self> // can write `Self` instead of `List<T>`
+  {
+    Nil,
+    Cons(T, Box<Self>) // likewise here
+  }
+  ```
+
+Compiler
+--------
+- [You can now mark traits with `#[must_use]`.][55663] Providing a warning if
```

Comment:
Since this is a language semantics change I'd move it up to the lang section.
This also doesn't seem to indicate what `#[must_use]` does when used on traits.

Reply:
The second sentence indicates what `must_use` does, but it's a sentence fragment. The period should be a comma.

Reply:
The second sentence doesn't actually indicate what it does tho. It's much too vague; the change that happened was that if you attach `#[must_use = "abc"]` to `trait Foo { .. }` then functions returning `-> impl Foo` will get `#[must_use = "abc"]` linting semantics. You should be able to understand this from the release notes. See https://github.com/rust-lang/rust/pull/55663/files#diff-3cd0617cc24bd130f52b9db16f414075 for an example.

Reply:
Ah, now I understand.